static/lookquran.js

```diff
@@ -11,6 +11,19 @@ $(function () {
 		$(".ayah-item").removeClass("playing");
 	};
 
+	initializerecitersList('ar');
+
+	function initializerecitersList(lang) {
+		$.getJSON("http://api.alquran.cloud/v1/edition?format=audio&language=" + lang, function (data) {
+			var recitersList = [];
+			$.each(data.data, function (key, val) {
+				recitersList.push(`<option value="${val.identifier}">${JSON.stringify(val.name).slice(1, -1)}</option>`);
```

Comment:
Can you use string concat instead of template string? as we haven't configured a transpiler yet.